core/src/com/unciv/logic/multiplayer/storage/UncivServerFileStorage.kt

```diff
@@ -76,6 +76,24 @@ object UncivServerFileStorage : FileStorage {
         return authenticated
     }
 
+    override fun checkAuthStatus(userId: String, password: String): AuthStatus {
+        var authStatus = AuthStatus.UNKNOWN
+        val preEncodedAuthValue = "$userId:$password"
+        authHeader = mapOf("Authorization" to "Basic ${Base64Coder.encodeString(preEncodedAuthValue)}")
+        SimpleHttp.sendGetRequest("$serverUrl/auth", timeout = timeout, header = authHeader) { success, result, code ->
+            if (success) {
+                authStatus = if (result.lowercase().contains("unregistered")) {
```

Comment:
This is not how I would expect unregistered to work, this should be an http 404 error, surely
Trusting textual results to convey success/failure in http is an antipattern

Reply:
"Acceptable but no data" sounds like http 204 to me (no content)